packages/mcp/mcp-client-chat/src/utils/zodSchema.ts

```
@@ -0,0 +1,320 @@
+
+  if (isNumber(schema.minLength)) {
+    stringSchema = stringSchema.min(schema.minLength);
+  }
+  if (isNumber(schema.maxLength)) {
+    stringSchema = stringSchema.max(schema.maxLength);
+  }
+
+  if (typeof schema.pattern === 'string') {
+    try {
+      stringSchema = stringSchema.regex(new RegExp(schema.pattern));
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used. Allowed unused caught errors must match /^_/u.

(@typescript-eslint/no-unused-vars)